project/settings/defaults.py

```diff
@@ -30,7 +30,7 @@
 )
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = bool(os.environ.get("DEBUG", False))
+DEBUG = True  # bool(os.environ.get("DEBUG", False))
```

Comment:
Usually you should just be able to edit this in `settings/local.py` and it'll override the default.

We should keep this as `bool(os.environ.get("DEBUG", False))` so that any env vars we pass are respected. Also, we should generally not have DEBUG on when deploying :)

Reply:
ACK I'm sorry about that, that was not supposed to be committed. It is a hack for my local setup where I couldn't get it respecting the darn environment variable for a while.